include/mbedtls/ssl_internal.h

```diff
@@ -741,6 +742,20 @@ struct mbedtls_ssl_handshake_params
 
 #endif /* MBEDTLS_SSL_SRV_C */
 
+        /* Incoming CertificateVerify */
+        struct
+        {
+            unsigned char verify_buffer[ 64 + 33 + 1 + MBEDTLS_MD_MAX_SIZE ];
```

Comment:
Minor: We should use macros instead of magic constants.

Reply:
(This is nicely explained further below)

Reply:
Used a macro here